icebergShaded/generate_iceberg_jars.py

```diff
@@ -64,12 +68,17 @@ def prepare_iceberg_source():
         shutil.rmtree(iceberg_src_dir_name, ignore_errors=True)
         run_cmd("git config user.email \"<>\"")
         run_cmd("git config user.name \"Anonymous\"")
-        run_cmd("git clone --branch %s https://github.com/apache/iceberg.git %s" %
-                (iceberg_src_branch_with_commit_hash, iceberg_src_dir_name))
+
+        # We just want the shallowest, smallest iceberg clone. We will check out the commit later.
+        run_cmd("git clone --depth 1 --branch %s https://github.com/apache/iceberg.git %s" %
+                (iceberg_src_branch, iceberg_src_dir_name))
 
     with WorkingDirectory(iceberg_src_dir):
         run_cmd("git config user.email \"<>\"")
```

Comment:
Is it possible to bypass this if already configured? It is causing issues like this:
![image](https://github.com/delta-io/delta/assets/1336227/d51a25d2-2ed6-4970-a0a4-c6813e123c99)